lib/python/compliance_checker_imos_files_config.json

```diff
@@ -1,47 +1,40 @@
 {
   "abos_da_ctd": {
-    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/eMII/test_files/ABOS/DA/IMOS_ABOS-DA_CSTZ_20180505T070800Z_EAC2000_FV01_EAC2000-2019-SBE37SM-RS232-1817_END-20190923T212000Z_C-20200422T015519Z.nc",
+    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/DWM/DA/EAC2000/CTD_timeseries/IMOS_DWM-DA_CSTZ_20180505T070800Z_EAC2000_FV01_EAC2000-2019-SBE37SM-RS232-1817_END-20190923T212000Z_C-20200422T015519Z.nc",
```

Comment:
As previously said, the reason test files were in a test_files directory is because as soon as the facility re-processes this particular file and uploads it, it will have a new file name (different creation date at the end) and this will break the unit test. Please copy this file, put it in test_files and refer to it there in the unit test.